internal/proto/authd/authd.proto

```diff
@@ -195,3 +195,31 @@ message ShadowEntry {
 message ShadowEntries {
   repeated ShadowEntry entries = 1;
 }
+
+service CLI {
```

Comment:
I wouldn't call it CLI but I rather something like Manager or something similar, as this something that we could ideally expose later in other tools (such as gcc), and we can include here every operation that is about the management of authd (or its users) itself, despite the UI that is used for that.

Reply:
Ok, what about this structure then:

```
service UsersService {
  rpc ListUsers(Empty) returns (Users);
  rpc DisableUser(DisableUserRequest) returns (Empty);
  rpc EnableUser(EnableUserRequest) returns (Empty);
}

message User {
  string name = 1;
  uint32 uid = 2;
  uint32 gid = 3;
  string gecos = 4;
  string homedir = 5;
  string shell = 6;
}

message Users {
  repeated User users = 1;
}

service GroupsService {
  rpc ListGroups(Empty) returns (Groups);
}

message Group {
  string name = 1;
  uint32 gid = 2;
  repeated string members = 3;
}

message Groups {
  repeated Group groups = 1;
}
```

Reply:
I implemented something similar to that